.eslintrc.js

```diff
@@ -17,7 +15,6 @@ module.exports = {
     sourceType: 'module'
   },
   plugins: [
-    'eslint-plugin-ante'
```

Comment:
I pushed up a new version of this plugin, so it should no longer complain about an ESLint incompatibility.

Reply:
The 'eslint-plugin-ante' thingy is back.